AmplifyTestCommon.podspec

```diff
@@ -23,7 +23,8 @@ Pod::Spec.new do |spec|
 
   spec.source_files = 'AmplifyTestCommon/**/*.swift'
   spec.dependency 'Amplify', '0.11.0'
-
+  spec.dependency 'AWSCore'
+  
```

Comment:
It's reasonable that we'd need this in AWSPluginsTestCommon, but we shouldn't put an AWSCore dep at the top level AmplifyTestCommon.

Reply:
Moved to AWSPluginsTestCommon